src/analyses/constant_propagator.h

```diff
@@ -195,17 +197,6 @@ class constant_propagator_ait:public ait<constant_propagator_domaint>
     replace(goto_model.goto_functions, ns);
   }
 
-  constant_propagator_ait(
-    goto_functionst::goto_functiont &goto_function,
-    const namespacet &ns,
-    should_track_valuet should_track_value = track_all_values):
-    dirty(goto_function),
-    should_track_value(should_track_value)
-  {
-    operator()(goto_function, ns);
-    replace(goto_function, ns);
-  }
```

Comment:
Why is this removed?  This seems unrelated to the change described in the commit message.

Reply:
I've put it back in, with a function identifier.